chsdi/templates/htmlpopup/sis_angaben.mako

```diff
@@ -9,7 +9,10 @@
     description = 'description_%s' % lang
     facility_kind = 'fackind_text_%s' % lang
     facility_status = 'facstatus_text_%s' % lang
-    datefrom = datetime.datetime.strptime(c['attributes']['valid_from'].strip(), "%Y%m%d").strftime("%d.%m.%Y")
+    try:
+        datefrom = datetime.datetime.strptime(c['attributes']['valid_from'].strip(), "%Y%m%d").strftime("%d.%m.%Y")
+    except:
+        datefrom = '-'
```

Comment:
Thanks but better test if this attr is None or not. Would be a
Slightly cleaner

Reply:
but less robust. db input for this function is a text column which can contain anything
```python
datefrom = datetime.datetime.strptime('anything_but_knickerbocker'.strip(), "%Y%m%d").strftime("%d.%m.%Y")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python2.7/_strptime.py", line 325, in _strptime
    (data_string, format))
ValueError: time data 'anything_but_knickerbocker' does not match format '%Y%m%d'

```